signer/src/storage/mod.rs

```diff
@@ -130,6 +131,12 @@ pub trait DbRead {
         &self,
     ) -> impl Future<Output = Result<Vec<model::Bytes>, Error>> + Send;
 
+    /// Get the outstanding signer UTXO
+    fn get_signer_utxo(
+        &self,
+        aggregate_key: &crate::keys::PublicKey,
```

Comment:
This functions should take the block hash of the bitcoin chain tip as an input.

Reply:
In addition to the `aggregate_key`, or only that? The issue description would suggest we "infer" (ie, pick all the valid) aggregate_key from the storage, but given it was present in the original fn signature I kept it.

Reply:
Oh yeah, I meant in addition to the `aggregate_key`.